<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkOdfMaximaExtractionViewControls</class>
 <widget class="QWidget" name="QmitkOdfMaximaExtractionViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QCommandLinkButton" name="m_ImportShCoeffs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Generate Q-Ball image and MITK compatible SH coefficient from other toolkits.</string>
     </property>
     <property name="text">
      <string>Start SH Coefficient Import</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>259</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="m_OutputVectorFieldBox">
        <property name="toolTip">
         <string>Only for visualization purposes! The vectors are automatically corrected for image spacing and for the largest eigenvalue in case of the tensor peak extraction.</string>
        </property>
        <property name="text">
         <string>Vector Field</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="m_OutputNumDirectionsBox">
        <property name="toolTip">
         <string>Output unsigned char image containing the number of directions per voxel.</string>
        </property>
        <property name="text">
         <string>#Peaks per Voxel</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="m_OutputDirectionImagesBox">
        <property name="toolTip">
         <string>4D image cotaining peak information</string>
        </property>
        <property name="text">
         <string>Peak Image</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Vector Normalization:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="m_NormalizationBox">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The vector fields are always coorected for image spacing and using the lagest eigenvalue in case of the tensor peak extraction. This is done for visualizytion purposes. The output direction images are not affected.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>No Normalization</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MAX Normalize</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Single Vec Normalization</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="m_CoeffImageFrame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Max. Peaks:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Relative Threshold:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="m_PeakThresholdBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Peak threshold relative to the largest peak per voxel.</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="m_AbsoluteThresholdBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Absolute peak threshold (only used for the finite differences method). The value is additionally scaled by 1/GFA.</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.030000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="m_MaxNumPeaksBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Maximum number of peaks to extract.</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Clustering Angle:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="m_ClusteringAngleBox">
           <property name="toolTip">
            <string>Cluster close directions. Define &quot;close&quot; here.</string>
           </property>
           <property name="maximum">
            <number>90</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Absolute Threshold:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Angular Threshold:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="m_AngularThreshold">
           <property name="toolTip">
            <string>Discard smaller peaks in the defined angle around the maximum peaks that were too far away to be clustered.</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>90</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="m_InputData">
     <property name="title">
      <string>Please Select Input Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="toolTip">
         <string>Select a tensor image or a SH coefficient image (generate using Q-Ball reconstruction view).</string>
        </property>
        <property name="text">
         <string>ShCoeff/DTI:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mask Image:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_ImageBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_MaskBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCommandLinkButton" name="m_StartPeakExtractionButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Extract ODF peaks using finite differences on the densely sampled ODF surface.</string>
     </property>
     <property name="text">
      <string>Start Peak Extraction</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Spherical Harmonic Convention</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QComboBox" name="m_ToolkitBox">
        <property name="toolTip">
         <string>Define SH coefficient convention (depends on toolkit)</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>MITK/FSL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MRtrix</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBoxWithSelectNone</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBoxWithSelectNone.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
